### ISSUE TITLE
issue summarizing a genomic database using a blank PROFILE.db

### ISSUE THREAD

**Issue by tdelmont** [merenlab/anvio#797]
using the stable version v4:

I have a genomic database processed with anvi'o v.4 without any metagenomic data associated to it, and I would like to take advantage of the processing capabilities of "anvi-summarize" to get basic info on each genome well organized.

I imported a collection using this command line:

---
anvi-import-collection -c DATA_CONTIGS.db contigs_basic_info.txt -C DATA --contigs-mode -p PROFILE.db/PROFILE.db
Num contigs in the contigs database ..........: 151,822
Num splits in the contigs database ...........: 175,662
Num contigs in the input file ................: 151,821
Num bins the input file describes ............: 810
Num contigs both input file and the db has ...: 151,821

* 151821 of 151821 contig names in the input file that matched to contig names in
the database resolved into 175661 split names that described in 810 bins.
Collections ..................................: The collection "DOMGD_v2" that describes 175,661 splits has been successfully added to the database at "PROFILE.db/PROFILE.db".
---

It seems this step worked fine.

One chalenge is that a PROFILE.db is required to run "anvi-summarize", so I created a blank PROFILE.db using this command line:

---
anvi-profile --blank-profile -c DATA_CONTIGS.db -o PROFILE.db --skip-hierarchical-clustering -S PROFILE
---

This too worked fine.

I then ran the following command line:

---
anvi-summarize -c DATA_CONTIGS.db -p PROFILE.db/PROFILE.db -C DATA -o SUMMARY_DATA
---

I got this error:

---
Contigs DB ...................................: Initialized: DOMGD_v2_CONTIGS.db (v. 10)
Auxiliary Data ...............................: Found: PROFILE.db/AUXILIARY-DATA.db (v. 2)
Profile Super ................................: Initialized with all 0 splits: PROFILE.db/PROFILE.db (v. 23)

WARNING
===============================================
Some of the bins in this collection (precisely 810 of 0 total) did not contain
any that appeared in the profile database. There are multiple reasons for why
this can happen. But one of the common scenario could be this: You imported an
external collection, and some of the bins you have in that collection contain a
small number of contigs that were too short to make it into the merged profile.
Well, if you would like to figure out what might be the scenario for your
experiment, here is the list of bin names that did not go through:
---

Basically, out of 810 genomes in my collection none could be processed.

Is that to be expected?

Note that alternatively, I recruited some metagenomic reads onto the genomic database, created a real PROFILE.db, and this way could process "anvi-summarize". However, this is not a logical way to proceed when one want only to work with a FASTA file without environmental signal. 

Thank you for considering this issue,

Tom

**Comment by meren** [comment]
Why is there no anvi'o version output? :)

**Comment by meren** [comment]
Oh, you mentioned that it is stable!

I think we fixed this in master, Tom :)

**Comment by tdelmont** [comment]
I see, for now let's close the issue then,

thanks
